src/database/migrations/V8__addDurationToAccessPass.ts

```diff
@@ -0,0 +1,21 @@
+import knex from 'knex';
+
+const ACCESS_PASS_TABLE = 'access_pass';
+
+export async function up(db: knex) {
+  await db.schema.table(ACCESS_PASS_TABLE, (table) => {
+    table.integer('duration');
+  });
+
+  await db(ACCESS_PASS_TABLE).where({ duration: null }).update({ duration: 60 });
+
+  await db.schema.alterTable(ACCESS_PASS_TABLE, (table) => {
+    table.integer('duration').notNullable().alter();
```

Comment:
you could do this on a table-default level if you'd like with `defaultTo` instead, but this is also nice

Reply:
Does that work retrospectively as well?

Reply:
it doesn't work retrospectively.